torch/library.py

```diff
@@ -555,18 +561,49 @@ def impl(qualname, types, func=None, *, lib=None):
     Examples:
         >>> import torch
         >>> import numpy as np
+        >>> # Example 1: Register function.
+        >>> # Define the operator
+        >>> torch.library.define("mylib::sin", "(Tensor x) -> Tensor")
+        >>>
+        >>> # Add implementations for the operator
```

Comment:
The new examples are confusing to me. I'd rather not have them in the examples section, or move all of the new examples to the bottom.

Reply:
Refactored so that Example 1 contains original example and Example 2 (at the bottom) contains new example). Also clarified the new example.